### PR TITLE
Search times

### PR DESCRIPTION
**Pull request by StefanieBier**
These are included in the AeD article and need to be in the content search limits as well

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/limits-for-content-search.md [hunk 1]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
 |Maximum body size in index  <br/> |67 million characters  <br/> |The total number of characters in the body of an email message and all its attachments. When an email message is indexed, all text in the body of the message and in all attachments is concatenated into a single string. The maximum size of this string that is indexed is 67 million characters.  <br/> |
 |Maximum unique tokens in body  <br/> |1 million  <br/> |As previously explained, tokens are the result of extracting text from content, removing punctuation and spaces, and then dividing it into words (called tokens) that are stored in the index. For example, the phrase  `"cat, mouse, bird, dog, dog"` contains 5 tokens. But only 4 of these are unique tokens. There is a limit of 1 million unique tokens per email message, which helps prevent the index from getting too large with random tokens.  <br/> |
 |||
+
+## Search Times
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
## Search times
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 2]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+Microsoft collects performance information for searches run by all organizations. While the complexity of the search query can impact search times, the biggest factor that affects how long searches take is the number of mailboxes searched. Although Microsoft doesn't provide a Service Level Agreement for search times, the following table lists average search times for collection searches based on the number of mailboxes included in the search.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
We collect performance information for searches run by all organizations. While the complexity of the search query can impact search times, the biggest factor that affects how long searches take is the number of mailboxes searched. Although we don't provide a Service Level Agreement for search times, the following table lists average search times for collection searches based on the number of mailboxes included in the search.
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 3]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+
+Number of mailboxes	Average search time
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
| Number of mailboxes| Average search time |
|:-----|:-----|
|100  <br/> |30 seconds<br/> |
|1,000  <br/> |45 seconds<br/> |
|10,000  <br/> |4 minutes<br/> |
|25,000  <br/> |10 minures<br/> |
|50,000  <br/> |20 minutes<br/> |
|100,000  <br/> |25 minutes<br/> |
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 4]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+100
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 5]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+30 seconds
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 6]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+1,000
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 7]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+45 seconds
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 8]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+10,000
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 9]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+4 minutes
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 10]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+25,000
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 11]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+10 minutes
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 12]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+50,000
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 13]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+20 minutes
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 14]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+100,000
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
```

File: microsoft-365/compliance/limits-for-content-search.md [hunk 15]
```diff
@@ -79,6 +79,23 @@ The following table describes the indexing limits that might result in an email
...
+25 minutes
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@StefanieBier

**Comment by markjjo** [comment]
@Yogeshmahi @JohanFreelancer9 These search times must be formatted as a table. At some point in this PR, the table formatting got removed. Table formatting needs to be added back into topic.

**Comment by JohanFreelancer9** [comment]
@StefanieBier - could you please format these times as a table, as requested? Thanks.

**Comment by markjjo** [comment]
@StefanieBier @JohanFreelancer9 I took care of the formatting and merged the PR; no more work is necessary

**Pull request merged**